templates/themes/tastefulplanet/pagelayout.html.twig

```diff
@@ -3,7 +3,7 @@
     <head>
         {% block seo_metas %}
             {% if location is defined and not location.isDraft %}
-{#                {% include "NovaeZSEOBundle::seometas_head.html.twig" %}#}
+{#                {% include "@NovaeZSEO/seometas_head.html.twig" %}#}
```

Comment:
Is there any reason why this is still here?

Reply:
My guess is we're waiting for release to be able to restore it, right @damianz5 @ciastektk?